x-pack/packages/ml/category_validator/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/ml-category-validator
+
+Empty package generated by @kbn/generate
```

Comment:
Would be good to have a brief description here.

Reply:
Ah, I missed that. Thanks 
Updated in https://github.com/elastic/kibana/pull/161261/commits/91de94b0d51cc5409db50d23de93670aa67da9ae